include/constants/flags.h

```diff
@@ -1352,9 +1352,9 @@
 #define FLAG_SYS_ENC_UP_ITEM                        (SYSTEM_FLAGS + 0x4D)
 #define FLAG_SYS_ENC_DOWN_ITEM                      (SYSTEM_FLAGS + 0x4E)
 #define FLAG_SYS_BRAILLE_DIG                        (SYSTEM_FLAGS + 0x4F)
-#define FLAG_SYS_BRAILLE_REGIROCK_HM                (SYSTEM_FLAGS + 0x50)
-#define FLAG_SYS_BRAILLE_REGICE_PUZZLE                       (SYSTEM_FLAGS + 0x51)
-#define FLAG_SYS_BRAILLE_REGISTEEL_HM               (SYSTEM_FLAGS + 0x52)
+#define FLAG_SYS_REGIROCK_PUZZLE_COMPLETE           (SYSTEM_FLAGS + 0x50)
```

Comment:
These three flag names are inconsistent. They should all be either COMPLETE or COMPLETED